### PR TITLE
Add a list of available boto profiles to the alton environment.

### PR DESCRIPTION
**Pull request by feanil**

### CODE REVIEW

File: playbooks/roles/alton/templates/alton_env.j2 [hunk 1]
```diff
@@ -6,3 +6,5 @@ export {{ name }}="{{ value }}"
 {% endif %}
 {%- endfor %}
 
+export BOTO_PROFILES="{{ ALTON_AWS_CREDENTIALS|join(';') }}"
```
Comment (jarv) [inline playbooks/roles/alton/templates/alton_env.j2:9]: `ALTON_AWS_CREDENTIALS` is a dictionary of dictionaries, does this work?

Comment (jarv) [inline playbooks/roles/alton/templates/alton_env.j2:9]: nm, it appears to work

**Comment by jarv** [comment]
after comments :+1:

File: playbooks/roles/alton/templates/alton_env.j2 (hunk 1 above)
Comment (jarv) [inline playbooks/roles/alton/templates/alton_env.j2:9]: personally I would have done something like `{{ ';'.join(ALTON_AWS_CREDENTIALS.keys()) }}`

Comment (feanil) [inline playbooks/roles/alton/templates/alton_env.j2:9]: I was just trying to stick to jinja syntax in there.

**Pull request merged**